### ISSUE TITLE
Accessibility: MAS20: alt text is "icon" for the images under "Extensions for the Visual Studio family of products"

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#59]
#A11yMAS
#MAS20
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Browser-Edge
Edge Version-Microsoft Edge 38.14393.461.0

**Repro Steps:**
1. Open https://marketplace.visualstudio.com/VSCode in EDGE
2. Press F12 key and check the alt text for the images  inside the "Extension Link" window available under "Featured", "Most Popular", "Trending" and "Recently Added" extension headers
**_Same issue is observed for the link:_** 
https://marketplace.visualstudio.com/items?itemName=donjayamanne.python [refer Python Screenshot]

**Actual Result:**
The alt text is "Icon" for all the images available inside the "Extension Link" window available under "Featured", "Most Popular", "Trending" and "Recently Added" extension headers 

**Expected Result:**
The alt text for the images available inside the "Extension Link" window available under "Featured", "Most Popular", "Trending" and "Recently Added" extension headers should be meaningful and not simply "Icon"

**User Impact:**
Users with memory and attention limitations benefit by being able to quickly rediscover their context after an interruption.

**Screenshots:**
![accessibility tree showing alt text for image is icon](https://cloud.githubusercontent.com/assets/24825441/21759589/6b50365c-d66b-11e6-985a-8c777e444c8b.png)
![alt text for images is icon](https://cloud.githubusercontent.com/assets/24825441/21759591/6b58ae04-d66b-11e6-97f8-c71130dac08c.png)
![alt text is icon for python image](https://cloud.githubusercontent.com/assets/24825441/21759611/91aef04a-d66b-11e6-92cb-3bf213b11f9e.png)

**Comment by waderyan** [comment]
This is a Marketplace issue.
